modules/packages/HDFS.chpl

```diff
@@ -495,7 +495,7 @@ module HDFS {
         var rc = hdfsPread(file.fs.hfs, file.hfile, offset, ptr, len:int(32));
         if rc == 0 {
           // end of file
-          return EEOF;
+          return EIO;
```

Comment:
For this and the changes in Curl.chpl I think it should just use `  private extern proc chpl_macro_int_EEOF():c_int;` and call that to get the error code. I don't think changing it to EIO will preserve the same behavior, but these modules aren't tested in regular testing, so you wouldn't see that in a regular test.